mannyhagman/react-fizz-buzz-demo
# Outline of application

## Purpose
This pull request 
- Address mobile layout inconsistencies 
- Adds a project README.md
- Disables non functioning inputs
## Testing Instructions
- Fork repository 
- `cd `into directory project root
- `npm install` or `yarn install` 
- Verify static site renders with 4 cards displaying the results of the FizzBuzz function

## Style and Lint check list
- [ √] This code has been tested
- [ √] This code has been linted 

## Potential Issues and Resolutions

- Inputs disabled while under development: fix coming in the next few days
- Validation for form has not been added
